subprojects/shipkit/src/main/groovy/org/shipkit/internal/gradle/versionupgrade/CiUpgradeDownstreamPlugin.java

```diff
@@ -22,10 +22,10 @@
 
     @Override
     public void apply(Project project) {
-        project.getPlugins().apply(CiReleasePlugin.class);
+        project.getRootProject().getPlugins().apply(CiReleasePlugin.class);
```

Comment:
Great! Can you update the Javadoc for the CiUpgradeDownstreamPlugin? @mockito/shipkit-developers, let's make sure we update the Javadocs when we're making changes. We need good javadocs for 1.0 ;)